src/js/ui-events.js

```diff
@@ -79,7 +79,7 @@ class UiEvents {
   _onInteractive() {
     this._offInteractive();
 
-    if (this._options.closeClass) {
+    if (this._options.closeClass && !this._options.useDefaultEvent) {
```

Comment:
This will break the `closeClass` option in case of thruthy `useDefaultEvent`.
The check for default action prevention should be within the event handler.

Reply:
done. Thank u was a remaining "test" code.. i just removed it